rest_framework/utils/throttling_duration_parser.py

```diff
@@ -0,0 +1,21 @@
+def parse_quantity_and_unit(quantity_unit_string):
+    """
+    Parse a combined quantity and unit string and return a dictionary containing the parsed values.
+
+    Args:
+        quantity_unit_string (str): A string that combines a numeric quantity and a unit, e.g., "5min", "10h".
+
+    Returns:
+        dict: A dictionary containing the parsed quantity and unit, with keys 'quantity' and 'unit'.
+              If the input string contains only a unit (e.g., "m"), quantity will be set to 1.
```

Comment:
This does not match any other docstring style in the project, it will just make confusion.

Reply:
I did not understand your point. Is it too much text ? or too long lines ?

Reply:
To put it simple: the part with `Args:` and `Returns:` are not used by any other comment in this repository. Keeping that here may cause confusion.